examples/readme.md

```diff
@@ -0,0 +1,151 @@
+|pytorch|[distributed MNIST (pytorch) using kubeflow](#mnist-kubeflow-pytorch)|[examples/trials/mnist-distributed-pytorch/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-distributed-pytorch)|
+|keras|[MNIST in keras](#mnist-keras)<br>[MNIST -- tuning with batch tuner](#mnist-batch)|[examples/trials/mnist-keras/](https://github.com/Microsoft/nni/tree/master/examples/trials/mnist-keras)<br>
+
+We also have some other examples :<br>
+<br><br>[CIFAR-10 examples](https://github.com/Microsoft/nni/tree/master/examples/trials/cifar10_pytorch):CIFAR-10 classification is a common benchmark problem in machine learning. The CIFAR-10 dataset is the collection of images. It is one of the most widely used datasets for machine learning research which contains 60,000 32x32 color images in 10 different classes. Thus, we use CIFAR-10 classification as an example to introduce NNI usage.
+<br><br>[Scikit-learn in NNI](https://github.com/Microsoft/nni/tree/master/examples/trials/sklearn):Scikit-learn is a pupular meachine learning tool for data mining and data analysis. It supports many kinds of meachine learning models like LinearRegression, LogisticRegression, DecisionTree, SVM etc. How to make the use of scikit-learn more efficiency is a valuable topic.NNI supports many kinds of tuning algorithms to search the best models and/or hyper-parameters for scikit-learn, and support many kinds of environments like local machine, remote servers and cloud.
+<br><br>[Automatic Model Architecture Search for Reading Comprehension](https://github.com/Microsoft/nni/tree/master/examples/trials/network_morphism):This example shows us how to use Genetic Algorithm to find good model architectures for Reading Comprehension.
+<br><br>[GBDT in nni](https://github.com/Microsoft/nni/tree/master/examples/trials/auto-gbdt):Gradient boosting is a machine learning technique for regression and classification problems, which produces a prediction model in the form of an ensemble of weak prediction models, typically decision trees. It builds the model in a stage-wise fashion as other boosting methods do, and it generalizes them by allowing optimization of an arbitrary differentiable loss function.
```

Comment:
"GBDT in nni" ---》 “GBDT in NNI”？ Due to you use "Scikit-learn in NNI" in line 44.